test/integration/test_kraken.py

```diff
@@ -113,8 +113,9 @@ def test_kraken_tool(tmpdir, kraken, kraken_db, input_bam):
     kraken.classify(input_bam, kraken_db, out)
     kraken.filter(out, kraken_db, out_filtered, 0.05)
     kraken.report(out_filtered, kraken_db, out_report)
+    with util.file.open_or_gzopen(out_filtered, 'r') as inf:
+        assert len(inf.read()) > 0
```

Comment:
Can you convert this call (`len(inf.read()) > 0`) to something that is O(1) memory and runtime (like `os.path.getsize`)? Especially memory. Also in `test_kraken`.